packages/neuron-ui/src/widgets/AlertDialog/index.tsx

```diff
@@ -51,7 +51,7 @@ const AlertDialog = ({
           v === 'cancel' ? (
             <Button type="cancel" onClick={onCancel} label={t('common.cancel')} />
           ) : (
-            <Button type="confirm" onClick={onCancel || onOk} label={t('common.confirm')} />
+            <Button type="confirm" onClick={onOk || onCancel} label={t('common.confirm')} />
```

Comment:
I haven't reviewed the previous PRs so I wonder why `onCancel` is bound to `confirm`

Reply:
When the type is `failed`, the `onCancel` function is used to close the dialog.

Reply:
Got it